--- setup.py	2018-05-31 17:13:27.577983986 -0400
+++ setup.py.travis	2018-05-31 17:22:50.112676822 -0400
@@ -24,7 +24,7 @@
     license='MIT',
     python_requires='>=3',
     packages=[__pkg_name__, __pkg_name__ + ".fonts"],
-    install_requires=['pygments', 'Pillow'],
+    install_requires=['pygments'],
     keywords='crossstitch cross stitch',
     entry_points={
         'console_scripts': [
